platform/core/src/measurements/classes/MeasurementApi.js

```diff
@@ -203,13 +204,14 @@ export default class MeasurementApi {
 
   retrieveMeasurements(patientId, timepointIds) {
     const retrievalFn = configuration.dataExchange.retrieve;
+    const { server } = this.options;
```

Comment:
Can you add JSDoc to `initialize`? I'm not sure what this method expects for options, and if it's using `server`, what properties `server` needs to have for it to be a valid option